### PR TITLE
[FIX] web: Many2ManyBinaryField was not functional

### PR DESCRIPTION
**Pull request by Iucapad**
Two commits:
- Fix the Many2ManyBinaryField 
- Delete FileDownloader component only used by 1 field

### CODE REVIEW

**Comment by Iucapad** [comment]
Note: Maybe we could also remove the FileUploader component and merge its behavior with FileInput component + replace in fields?

**Comment by Iucapad** [comment]
@aab-odoo Hey, can you take a look at this PR?
The first commit fixes the Many2ManyBinary field + tests
The second one remove an unnecessary component only used once in a template (I created this component initially but it is not needed)

File: addons/web/static/src/views/fields/many2many_binary/many2many_binary_field.xml [hunk 1]
```diff
@@ -4,22 +4,22 @@
...
             </div>
             <div t-if="!props.readonly" class="oe_add">
-                <FileUploader
+                <FileInput
                     acceptedFileExtensions="props.acceptedFileExtensions"
                     multiUpload="true"
-                    onUploaded.bind="onFileUploaded"
+                    onUpload.bind="onFileUploaded"
+                    resModel="'ir.attachment'"
+                    resId="props.record.resId or props.record.context.uid"
```
Comment (Iucapad) [inline addons/web/static/src/views/fields/many2many_binary/many2many_binary_field.xml:17]: @aab-odoo Does it sounds right to you? It no longer crashes with this. I'm not sure about what the resId must be for the FileInput

**Comment by Iucapad** [comment]
@aab-odoo It should be ready to merge

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Pull request closed without merging**